src/core/filesystem.cc

```diff
@@ -1753,7 +1759,10 @@ GetFileSystem(const std::string& path, FileSystem** file_system)
         "-DTRITON_ENABLE_S3=ON.");
 #else
     Aws::SDKOptions options;
-    Aws::InitAPI(options);
```

Comment:
c++11 call_once is probably a better, thread-safe way to make sure InitAPI is only called once.

Reply:
Good point. Fixed 👍